Added zh_cn.json

Mysitia opened the pull request:
Added Simplified Chinese localization.

MehdiNoui commented:
HI ! Thank you for contributing for this mod. Could you please update it based on my latest push ? I've added quite a few things and I want to include this translation before the next update. Thank you so much !

Mysitia commented:
> HI ! Thank you for contributing for this mod. Could you please update it based on my latest push ? I've added quite a few things and I want to include this translation before the next update. Thank you so much !

I have updated it according to the latest push, I'm glad to be able to help you.I absolutely love your mod (for Minecraft), and I'm really looking forward to future updates!

MehdiNoui commented:
Thank you for your motivation and for your contributions. It really means so much to me that my humble work is getting love and that there are people out there that has been enjoying it.
